presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -341,6 +344,55 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+S3SelectPushdown
+^^^^^^^^^^^^^^^^
+S3SelectPushdown enables pushing down projection (SELECT) and predicate (WHERE) processing to `S3 Select <https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html>`_. With S3SelectPushdown
```

Comment:
Add a newline after section headers.

We wrap text in documentation at around 80 characters to improve readability. Notice how GitHub soft-wraps these lines in the PR, which means they are too long. (The S3 encryption docs above are a bad example since they are also too long. The other sections in this file are better examples.)